js_modules/dagster-ui/packages/ui-core/src/integrations/IntegrationIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import {Box, IconWrapper} from '@dagster-io/ui-components';
+import {Spacing} from '@dagster-io/ui-components/src/components/types';
+import styled from 'styled-components';
+
+import {KNOWN_TAGS, KnownTagType, extractIconSrc} from '../graph/OpTags';
```

Comment:
I don't actually know if `OpTags` is an appropriate source of truth for these icons.

Reply:
Perhaps we can standardize, but this is what we use for docs.

https://github.com/dagster-io/dagster/tree/master/docs/static/images/integrations

Reply:
Ah excellent, we should definitely unify these.

Reply:
I'll come back to this once we're pulling in actual integration page data.

Reply:
Sounds good. This also has me thinking that we might end up embedding your marketplace UI in docs itself. But we'll talk about that more another time, heh.